docs/extensions/operatingsystemconfig.md

```diff
@@ -42,6 +42,18 @@ The high-level flow is as follows:
 
 1. After the VM has been provisioned the `downloader` script starts and fetches the appropriate `Secret` for its worker pool (containing the "original" user-data) and applies it.
 
+### Detailed bootstrap flow with the feature gate `BootstrapTokenForVMs` enabled
+
+If the featureFlag `BootstrapTokenForVMs` is enabled a file with the content "<<BOOTSTRAP_TOKEN>>" is added to the operatingsystem-config.
+It is passed to the worker controller (here MCM).
```

Comment:
MCM is not the worker controller.

Reply:
Got it. The MCM is generated by our worker controllers.